command/api/token/create.go

```diff
@@ -0,0 +1,133 @@
+package token
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/google/uuid"
+	"github.com/urfave/cli"
+	"go.step.sm/cli-utils/errs"
+	"go.step.sm/cli-utils/ui"
+)
+
+func createCommand() cli.Command {
+	return cli.Command{
+		Name:   "create",
+		Action: cli.ActionFunc(createAction),
+		Usage:  "create a new token",
+		UsageText: `**step api token create** <team-id> <crt-file> <key-file>
+[**--api-url**=<url>]
+`,
+		Flags: []cli.Flag{
+			apiURLFlag,
+		},
+		Description: `**step ca api token create** creates a new token for connecting to the Smallstep API.
+
+## POSITIONAL ARGUMENTS
+
+<team-id>
+:  UUID of the team the API token will be issued for. This is available in the Smallstep dashboard.
+
+<crt-file>
+:  File to read the certificate (PEM format). This certificate must be signed by a trusted root configured in the Smallstep dashboard.
+
+<key-file>
+:  File to read the private key (PEM format).
+
+## Examples
+
+$ step api token create ff98be70-7cc3-4df5-a5db-37f5d3c96e23 internal.crt internal.key
+`,
+	}
+}
+
+type createTokenReq struct {
+	TeamID string   `json:"teamID"`
+	Bundle [][]byte `json:"bundle"`
+}
+
+type createTokenResp struct {
+	Token   string `json:"token"`
+	Message string `json:"message"`
+}
+
+func createAction(ctx *cli.Context) (err error) {
+	if err := errs.NumberOfArguments(ctx, 3); err != nil {
+		return err
+	}
+
+	args := ctx.Args()
+
+	teamID := args.Get(0)
+	crtFile := args.Get(1)
+	keyFile := args.Get(2)
```

Comment:
It might be nice to support retrieving the key from a KMS, but I think supporting just the file is sufficient atm.